aioftp/client.py

```diff
@@ -491,6 +495,13 @@ class Client(BaseClient):
 
     :param encoding: encoding to use for convertion strings to bytes
     :type encoding: :py:class:`str`
+
+    :param ssl: if given and not false, a SSL/TLS transport is created
```

Comment:
please add link to https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.create_connection

Reply:
done